### ISSUE TITLE
installation instructions fail due to version of "python3-chewie"

### ISSUE THREAD

**Issue by blp** [faucetsdn/faucet#2399]
I ran through the installation instructions at https://docs.faucet.nz/en/latest/tutorials/first_time.html, by installing the base system for Ubuntu 16.04 and running the commands under "Package Installation".  It worked OK up to "sudo apt-get install faucet-all-in-one", which yields the following error:

> Reading package lists... Done
> Building dependency tree       
> Reading state information... Done
> Some packages could not be installed. This may mean that you have
> requested an impossible situation or if you are using the unstable
> distribution that some required packages have not yet been created
> or been moved out of Incoming.
> The following information may help to resolve the situation:
> 
> The following packages have unmet dependencies:
>  faucet-all-in-one : Depends: faucet (>= 1.7.0) but it is not going to be installed
>                      Depends: gauge (>= 1.7.0) but it is not going to be installed
>                      Depends: python3-faucet (>= 1.7.0) but it is not going to be installed
> E: Unable to correct problems, you have held broken packages.

I tried installing just "faucet":

> $ sudo apt-get install faucet
> Reading package lists... Done
> Building dependency tree       
> Reading state information... Done
> Some packages could not be installed. This may mean that you have
> requested an impossible situation or if you are using the unstable
> distribution that some required packages have not yet been created
> or been moved out of Incoming.
> The following information may help to resolve the situation:
> 
> The following packages have unmet dependencies:
>  faucet : Depends: python3-faucet (>= 1.7.0) but it is not going to be installed
> E: Unable to correct problems, you have held broken packages.

When I drilled down far enough, I found what appears to be the root cause:
>  python3-faucet : Depends: python3-chewie (< 0.0.3) but 0.0.5 is to be installed

I guess that Faucet is going to be hard to install if it can't be compatible with the latest version of this python package.

(I also tried installing Debian stretch and Debian testing.  Same result.)

**Comment by gizmoguy** [comment]
Sorry about this @blp, we're currently trying to tame our debian packaging so that it is reproducible and requires as little manual intervention as possible. Sadly we've still not got it quite right yet as you found.

On Friday we have released a new version of our 802.1x library (chewie) in preparation for tomorrow's faucet release (1.8.18). This new package of chewie accidentally (which is not compatible with older faucet versions) ended up in our official faucet debian repo 2 days early because I wasn't paying attention and some of my poor CI code pushed it automatically.

I have removed the bad packages for now so that faucet is installable again and assigned a couple new bugs (#2400 and #2401) to myself to ensure this doesn't happen again.

Sorry for the inconvenience and happy fauceting.
